README.md

```diff
@@ -0,0 +1,38 @@
+# Environment Validator
+
+Validates that the node engine being used and the modules installed match those
+described in your `package.json`. This is helpful when multiple people are
+working on a project and you want to be sure that their node environments are
+in sync.
+
+## Getting started
+
+1. Include this module in your `package.json`
+2. `npm install`
```

Comment:
could also just specify `npm i environment_validator --save` to automatically install it and add it to the dependencies section in `package.json`